scripts/package.json

```diff
@@ -9,8 +9,6 @@
   "author": "Firebase",
   "license": "MIT",
   "dependencies": {
-    "firebase": "^3.5.0",
-    "firebase-admin": "latest",
-    "firebase-functions": "latest"
+    "firebase": "^3.5.0"
```

Comment:
is this `package.json` file actually required? If the script is run `node scripts/test-functions-deploy.js`, and `firebase` is already a dev dependency of the package, then do we need to keep this file around?